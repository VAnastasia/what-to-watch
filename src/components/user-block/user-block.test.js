import React from "react";
import renderer from "react-test-renderer";
import UserBlock from "./user-block.jsx";

it(`UserBlock component render correctly`, () => {
  const tree = renderer.create(
      <UserBlock
        isAuth={true}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
