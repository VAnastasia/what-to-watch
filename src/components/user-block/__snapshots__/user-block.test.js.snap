// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserBlock component render correctly 1`] = `
<div
  className="user-block"
>
  <div
    className="user-block__avatar"
  >
    <img
      alt="User avatar"
      height="63"
      src="avatar"
      width="63"
    />
  </div>
</div>
`;
